Modules/socketmodule.h

```diff
@@ -3,22 +3,22 @@
 /* Includes needed for the sockaddr_* symbols below */
 #ifndef MS_WINDOWS
 #ifdef __VMS
-#   include <socket.h>
```

Comment:
Can we have the change without reformatting this entire file?

Reply:
Oops sorry , will revert that commit and just add the comment only.